src/MSBuild.Sdk.SqlProj.Templates/templates/sqlproj/sqlproj.csproj

```diff
@@ -1,11 +1,17 @@
 <Project Sdk="MSBuild.Sdk.SqlProj/#{NBGV_NuGetPackageVersion}#">
     <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
+        <TargetFramework>netstandard2.1</TargetFramework>
         <SqlServerVersion>#{SqlServerVersion}</SqlServerVersion>
         <RunSqlCodeAnalysis>#{CodeAnalysis}</RunSqlCodeAnalysis>
```

Comment:
I know this hasn't changed, but now looking at this I'm thinking that this should perhaps be defaulted in `Sdk.props` with a condition that it hasn't been set in the project?

Reply:
@jmezach Could you help we with that? 🙏 (Clueless with MsBuild as usual)

Reply:
Something like `<RunSqlCodeAnalysis Condition="'$(RunSqlCodeAnalysis)' == ''">$(CodeAnalysis)</RunSqlCodeAnalysis>` would probably do the trick ;).

Reply:
What does `$(CodeAnalysis)` refer to?

Reply:
@jmezach I tried with:

`<RunSqlCodeAnalysis Condition="'$(RunSqlCodeAnalysis)' == ''">True</RunSqlCodeAnalysis>`

Reply:
;) To be honest, I don't know what $(CodeAnalysis) refers to, but that is what we were using in the template as well right?

Reply:
@jmezach That is a template parameter, that you can override when creating the new project file using "dotnet new"

Reply:
Oh, sorry, I read that as an MSBuild property. I think it makes sense to be able to turn it on/off from the template, so I guess we don't need to change anything here ;).

Reply:
Got it! I will revert my latest change then

Reply:
It is defined here: "src\MSBuild.Sdk.SqlProj.Templates\templates\sqlproj\.template.config\template.json"